torchvision/ops/poly_loss.py

```diff
@@ -0,0 +1,66 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+from ..utils import _log_api_usage_once
+
+
+def poly_loss(
+    x: Tensor,
+    target: Tensor,
+    eps: float = 2.0,
+    weight: Optional[Tensor] = None,
+    ignore_index: int = -100,
```

Comment:
this is just to have default ignore as false right? Why not set it to None and check if its not None?

Reply:
My understanding is he is following very closely the default values of established losses in Core ([example](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html)). I think that's a good idea.

Reply:
aah I didn't know about this. Nice

Reply:
Yup that's what I was going for